### PR TITLE
Update README files using the url placeholder for elastic documentation links

### PR DESCRIPTION
**Pull request by mrodm**
## What does this PR do?

Update README files to use the new `{{ url <key> <link> }}` notation introduced in https://github.com/elastic/elastic-package/pull/938

To accomplish that, a new file in the root of the repository `links_table.yml` has been added with all the links to elastic documentation found in the current packages.

In general, there are no changes in README contents with this PR. There are some changes due to some markdown syntax issues.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- ~[ ] I have verified that all data streams collect metrics or logs.~
- ~[ ] I have added an entry to my package's `changelog.yml` file.~
- ~[ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).~

## How to test this PR locally

- In elastic-package repository, Build elastic-package command from the branch in the PR (`make build`)
- In integrations repository, go to any of the packages updated and run `elastic-package build`. This command should generate the README without failures.

## Author's Checklist

- [x] Requires a new version of elastic-package integrated including https://github.com/elastic/elastic-package/issues/948

## Related issues

- Requires https://github.com/elastic/elastic-package/issues/948
- Closes https://github.com/elastic/integrations/issues/4067
- Part of https://github.com/elastic/integrations/issues/2652

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -24,6 +24,9 @@ pipeline {
     JOB_GCS_EXT_CREDENTIALS = 'beats-ci-gcs-plugin-file-credentials'
     STACK_VERSION = "${params.stackVersion}"
 
+    // Elastic package links definitions
+    ELASTIC_PACKAGE_LINKS_FILE_PATH = "${env.HOME}/${env.BASE_DIR}/links_table.yml"
+
```
Comment (mrodm) [inline .ci/Jenkinsfile:29]: By default, `elastic-package` would use the file at the root of the directory. But, I would prefer to keep it explicitly this path.

**Review by efd6 — APPROVED** [review]
Approving for the SEI-owned files.

Is there testing in `elastic-package` for use of links that are included in the `links_table.yml` file to prevent them flowing back in to markdown files?

**Review by gizas — APPROVED** [review]

**Review by mtojek — APPROVED** [review]
LGTM

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Review by zmoog — APPROVED** [review]
LGTM, from cloud monitoring.

**Review by crespocarlos — APPROVED** [review]
LGTM! from Infra Monitoring UI. Tested it following the test steps successfully. elastic-package 0.61 still doesn't support this.

**Comment by mrodm** [comment]
> LGTM! from Infra Monitoring UI. Tested it following the test steps successfully. elastic-package 0.61 still doesn't support this.

Still not released a new version with these changes. This will be included in the next one (0.62.0)

**Pull request merged**